scripts/gen_man_pages.sh

```diff
@@ -2,23 +2,37 @@
 
 # Usage: ./gen_man_pages.sh DESTDIR PREFIX
 
+# Create given directory if not exist.
+create_directory_if_not_exist() {
+    if [ ! -d "$1" ]; then
+        mkdir -p "$1"
+    fi
+}
+
+# Check if lncli is installed.
+check_lncli_installed() {
+    if ! command -v lncli &> /dev/null
+    then
+        echo "lncli could not be found. Please install lncli before running this script."
+        exit 1
+    fi
+}
+
 DESTDIR="$1"
 PREFIX="$2"
+FULLPATH="$DESTDIR$PREFIX"
 
-# Check if lncli is installed.
-if ! command -v lncli &> /dev/null
-then
-    echo "lncli could not be found. Please install lncli before running this script."
-    exit 1
-fi
+check_lncli_installed
 
 # Ignore warnings regarding HTMLBlock detection in go-md2man package
 # since using "<...>" is part of our docs.
 lncli generatemanpage 2>&1 | grep -v "go-md2man does not handle node type HTMLSpan" || true
 
-echo "Installing man pages to $DESTDIR$PREFIX/share/man/man1."
-install -m 644 lnd.1 "$DESTDIR$PREFIX/share/man/man1/lnd.1"
-install -m 644 lncli.1 "$DESTDIR$PREFIX/share/man/man1/lncli.1"
+create_directory_if_not_exist "$FULLPATH/share/man/man1"
```

Comment:
I think we can directly do `mkdir -p "$FULLPATH/share/man/man1"` here, as it doesn't fail if the directory already exists.